wrangler-core/src/main/java/io/cdap/wrangler/statistics/BasicStatistics.java

```diff
@@ -55,10 +58,15 @@ public Row aggregate(List<Row> rows) {
           if (value.isEmpty()) {
             stats.increment(column, "empty");
           } else {
+            // this call is very expensive for string > 2000 characters, took seconds to return
             Map<String, List<String>> finds = engine.findWithType(value);
             for (String find : finds.keySet()) {
               types.increment(column, find);
             }
+            // TODO: this is a workaround for CDAP-18262, to proper fix we should revisit this computation logic
```

Comment:
file a TODO jira for this ? or just leave CDAP-18262 open and mark fix version as 6.6.0 ?

Reply:
I will file a new JIRA for this and close 18262

Reply:
https://cdap.atlassian.net/browse/CDAP-18277